### DISCUSSION TITLE
docs: progress bar content enhancements

### DISCUSSION THREAD

**Discussion by laurenmrice** [carbon-design-system/carbon#15176] [Ideas]
## Content enhancements

- The current progress bar component usage and style guidance has been reviewed from accessibility with requested updates.

---

- Suggestions for content enhancements are provided in pdfs below:
   - Usage tab guidance: https://ibm.ent.box.com/file/953648079387?s=jvu3n241qwsynpuoit2cb6hq2byx3p7g
   - Style tab guidance:  https://ibm.box.com/s/xjt4n00xi71tijvaj0fsdbiw0gtau942
   - Images with the success state need to be updated to used the `$support-success` color token.

**Comment by thyhmdo** [comment]
**Changes for Usage tab**
- [x] Updated Code and Accessibility tabs
- [x] Overview: changed loading spinners to loading
- [x] Variants : Updated the content
- [x] Determined progress bar: Updated the content
- [x] Anatomy: Updated Anatomy image and content
- [x] Alignment: Alignment to Text alignment; updated the content
- [x] Content: Updated labels content
- [x] Universal behaviors: Add image states; Updated the content and image for Error state
- [x] Accessibility: Updated the content

**Changes for Style tab**
- [x] Omit theme images
